server/db/index.js

```diff
@@ -1,35 +1,64 @@
 const db = require('./db')
-const { User, Snippet, Stretch, Comment } = require('./models')
+const models = require('./models')
+const {
+  User,
+  StretchAnswer,
+  Stretch,
+  Comment,
+  Cohort,
+  CohortStretch,
+  Category,
+  CohortUser
+} = models
 
 function initDb(force = false) {
   return db.authenticate().then(() => {
     // Sequelize associations
     // --------------------------
-    // A user has many snippets
-    Snippet.belongsTo(User)
-    User.hasMany(Snippet)
 
-    // A stretch has many snippets
-    Snippet.belongsTo(Stretch)
-    Stretch.hasMany(Snippet)
+    // Many-to-Many between User and Cohort
+    CohortUser.belongsTo(Cohort)
```

Comment:
I think I'm still not totally clear on why a many-to-many relationship buys us here. Why can't a user just have a cohortId? A user shouldn't be able to belong to many cohorts, should they?